src/Sulu/Bundle/ContactBundle/Resources/js/containers/ContactAccountSelection/ContactAccountSelection.js

```diff
@@ -0,0 +1,161 @@
+// @flow
+import React, {Fragment} from 'react';
+import {observer} from 'mobx-react';
+import {action, computed, observable, toJS} from 'mobx';
+import equals from 'fast-deep-equal';
+import {MultiItemSelection} from 'sulu-admin-bundle/components';
+import {MultiListOverlay} from 'sulu-admin-bundle/containers';
+import {translate} from 'sulu-admin-bundle/utils';
+import ContactAccountSelectionStore from './stores/ContactAccountSelectionStore';
+import contactAccountSelectionStyles from './contactAccountSelection.scss';
+
+// TODO extract into separate file?
```

Comment:
These prefixes are need in this file and in the `ContactAccountSelectionStore`, should we extract them into a separate `constants.js` file? Or are there any other ideas?